ui/main/package.json

```diff
@@ -56,6 +57,7 @@
     "ng-event-source": "1.0.14",
     "ngx-chips": "3.0.0",
     "ol": "6.5.0",
+    "rrule": "^2.7.1",
```

Comment:
Do not use ^ for version number as it does guarantee the version (we have to keep upgrades under control wuht precise version of dependencies)

Reply:
Done.